high-level-tests/e2e/cypress/support/step_definitions/user-tutorials.js

```diff
@@ -1,13 +1,13 @@
 when(`j‘enregistre le tutoriel {string}`, (tutorialName) => {
-  cy.contains('.tutorial__content', tutorialName).find('.tutorial-content__save-tutorial').click();
+  cy.contains('.tutorial__content', tutorialName).find('.tutorial-content-actions__save').click();
```

Comment:
Dans un test e2e je trouve plus intéressant de s'éloigner au maximum de l'implémentation pour rendre le test le plus robuste possible. Les classes CSS sont susceptible de changer sans pour autant altérer le comportement de l'application. Grâce à `contains` on peut venir sélectionner les éléments par leur contenu. Je suggère donc d'utiliser le contenu du button pour le sélectionner (ce n'est peut être pas possible à cause de l'icône, je n'ai pas testé) : 

```suggestion
  cy.contains('.tutorial__content', tutorialName).contains('Enregistrer').click();
```

Reply:
Bien vu !